tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -135,6 +135,26 @@ spec:
       description: Number of parallel workers to use for policy evaluation.
       default: "1"
 
+    - name: SINGLE_COMPONENT
+      description: Reduce the Snapshot to only the component whose build caused the Snapshot to be created
+      type: string
+      default: "false"
+
+    - name: CUSTOM_RESOURCE
```

Comment:
The name and description of this parameter, and the one below, should reflect that is associated with `SINGLE_COMPONENT` mode. Consider:

```yaml
- name: SINGLE_COMPONENT_NAME
  description: >
    Name, including kind, of the Kubernetes resource to query for labels when single
    component mode is enabled, e.g. pr/somepipeline.
- name: SINGLE_COMPONENT_NAMESPACE
  description: >
    Kubernetes namespace where the SINGLE_COMPONENT_NAME is found. Only used 
    when single component mode is enabled.
```

Reply:
Agreed.

Reply:
Maybe `SINGLE_COMPONENT_CR` and `SINGLE_COMPONENT_NS`.

Reply:
Is it ever not a pipeline run? Maybe SINGLE_COMPONENT_PR ?

Reply:
Since you need to specify the kind, the implication is it might be something other than a PipelineRun. Is that the intention?

Reply:
If we have to change it, then I would prefer `SINGLE_COMPONENT_CUSTOM_RESOURCE` and `SINGLE_COMPONENT_CUSTOM_RESOURCE_NAMESPACE` 

> Is it ever not a pipeline run? Maybe SINGLE_COMPONENT_PR ?

Yes in Release Pipelines, it is a `Snapshot`

Reply:
> I would prefer `SINGLE_COMPONENT_CUSTOM_RESOURCE` and `SINGLE_COMPONENT_CUSTOM_RESOURCE_NAMESPACE`

Works for me.

Reply:
It's quite long I guess. `SINGLE_COMPONENT_CUSTOM_RESOURCE_NS` maybe?

Reply:
I am ok with `SINGLE_COMPONENT_CUSTOM_RESOURCE` and `SINGLE_COMPONENT_CUSTOM_RESOURCE_NS`